src/main/resources/de/theit/jenkins/crowd/CrowdSecurityRealm/CacheConfiguration/config.jelly

```diff
@@ -31,4 +31,7 @@ THE SOFTWARE.
     <f:entry field="ttl" title="${%crowd.cache.TTL}">
         <f:select default="300"/>
     </f:entry>
+    <f:entry title="${%crowd.useTokenCache}" field="useTokenCache">
+        <f:checkbox />
+    </f:entry>    
```

Comment:
Why do you want to add it as an additional parameter?
For me i should be turned on or off like other caches.

Reply:
Thanks for the quick response. The reason I thought this should be separate is because while other caches create `delayed propagation of user changes`, enabling this option will delay even the user being disabled or the Crowd application being disabled from being enforced for the duration of the Cache TTL. This may or may not be acceptable to all users. Let me know if you still think cache should be a singular option.

Reply:
Ok, so let's leave it as you suggested.
Please only add this reasoning to the parameter helper.